src/core/lib/iomgr/ev_epollsig_linux.c

```diff
@@ -1021,10 +1021,11 @@ static void push_front_worker(grpc_pollset *p, grpc_pollset_worker *worker) {
 }
 
 /* p->mu must be held before calling this function */
-static grpc_error *pollset_kick(grpc_pollset *p,
+static grpc_error *pollset_kick(grpc_exec_ctx *exec_ctx, grpc_pollset *p,
                                 grpc_pollset_worker *specific_worker) {
   GPR_TIMER_BEGIN("pollset_kick", 0);
   grpc_error *error = GRPC_ERROR_NONE;
+  GRPC_STATS_INC_POLLSET_KICK(exec_ctx);
```

Comment:
Move this into `pollset_worker_kick()` , right before `pthread_kill()`

Reply:
Also, I know you didn't add other counters here but atleast `GRPC_STATS_INC_POLLSET_KICKED_WITHOUT_POLLER` would be useful for comparison

Reply:
See above comment, I think this is the right place